docs/supported-sources/github.md

```diff
@@ -0,0 +1,46 @@
+# GitHub
+
+[GitHub](https://github.com/) is a developer platform that allows developers to create, store, manage and share their code.
+
+ingestr supports GitHub as a source.
+
+## URI format
+
+The URI format for GitHub is as follows:
+
+```plaintext
+github://?access_token=<access_token>&owner=<owner>&name=<name>
+```
+
+URI parameters:
+
+- `access_token`: Access Token used for authentication with the GitHub API
+- `owner`: Refers to the owner of the repository
+- `name`: Refers to the name of the repository
```

Comment:
I'd suggest naming this as `repo`, since `name` is not very explanatory for various resources we might add under this source.